.github/workflows/release.yml

```diff
@@ -1,124 +1,251 @@
+        with:
+          minikube version: 'v1.17.1'
+          kubernetes version: 'v1.19.2'
+      - name: Login to minikube docker registry
+        run: eval $(minikube -p minikube docker-env)
+      - name: Build and Publish image
+        uses: docker/build-push-action@v2
+        with:
+          builder: ${{ steps.buildx.outputs.name }}
+          context: .
+          load: true
+          file: ./Dockerfile
+          tags: |
+            ${{ needs.preconditions.outputs.repo_name }}:latest
+          labels: |
+            org.opencontainers.image.title=${{ needs.preconditions.outputs.repo_name }}
+            org.opencontainers.image.description=${{ github.event.repository.description }}
+            org.opencontainers.image.source=${{ github.event.repository.html_url }}
+            org.opencontainers.image.url=${{ github.event.repository.html_url }}
+            org.opencontainers.image.revision=${{ github.sha }}
+      - name: Run chart-testing (install)
+        run: ct install  --charts helm/${{ needs.preconditions.outputs.repo_name }}/ --chart-repos vitalam-node=https://digicatapult.github.io/vitalam-node/,vitalam-ipfs=https://digicatapult.github.io/vitalam-ipfs/
 
-  # Github releases are only performed when the release version changes
-  build-release:
-    name: Build release
+  lint:
+    name: Run lint
     runs-on: ubuntu-latest
-    needs: [build-docker, get-version]
-    if: ${{ needs.get-version.outputs.is_new_version == 'true' }}
+    needs: [preconditions]
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@master
+        with:
+          node-version: 16.x
+      - name: Use npm v8
+        run: npm install -g npm@8
+      - name: Cache Node.js modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - name: Install Packages
+        run: npm ci
+      - name: Lint
+        run: npm run lint
 
+  tests:
+    name: Run tests
+    runs-on: ubuntu-latest
+    needs: [preconditions]
     steps:
       - uses: actions/checkout@master
+      - uses: actions/setup-node@master
+        with:
+          node-version: 16.x
+      - name: Use npm v8
+        run: npm install -g npm@8
+      - name: Cache Node.js modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.OS }}-node-
+            ${{ runner.OS }}-
+      - name: Install Packages
+        run: npm ci
+      - name: Setup dependencies
+        run: docker-compose up -d
+      - name: Sleep
+        uses: kibertoad/wait-action@1.0.1
+        with:
+          time: '30s'
```

Comment:
why sleep?

Reply:
Because otherwise we'd bring up the deps and it would immediately start running tests against an uninitiated blockchain node which wouldn't respond.

Reply:
figured it would be something like that